nautobot/utilities/testing/views.py

```diff
@@ -461,6 +463,28 @@ def test_create_object_with_constrained_permission(self):
             else:
                 self.assertInstanceEqual(self._get_queryset().last(), self.form_data)
 
+        def test_slug_autocreation(self):
+            """Test that slug is autocreated through ORM."""
+            # This really should go on a models test page, but we don't have test structures for models.
```

Comment:
It would be **nice** to have a base class for model testing, even if it only contained these two test cases to start with. Maybe we can open a `housekeeping` item for that.

Reply:
Sure, `Housekeeping` would be nice, if you don't mind.